lib/AcqCheckboxFilter/AcqCheckboxFilter.js

```diff
@@ -21,6 +21,11 @@ const AcqCheckboxFilter = ({
   onChange,
   options = [],
 }) => {
+  const dataOptions = options.map(({ label, labelId: intlLabelId, ...restOption }) => ({
```

Comment:
what is the reason of `intlLabelId` ?

Reply:
`labelId` is a prop of `AcqCheckboxFilter`

Reply:
shouldn't it be `label: intlLabelId ? <FormattedMessage id={intlLabelId} /> : label,` then?

Reply:
Sure) thnx